context/app/static/js/pages/Publication/Publication.jsx

```diff
@@ -25,6 +25,7 @@ function Publication({ publication, vignette_json }) {
     sub_status,
     doi_url,
     contributors = [],
+    contacts = [],
```

Comment:
Were the need for these defaults confirmed in your work? Do certain publications have either no `contributors` or no `contacts`?

Reply:
I haven't encountered a case of a publication without a `contacts` array - was mainly assigning a default to follow the convention for `contributors`. This could be removed, as a default empty array is also given in `ContributorsTable`.